package_control/automatic_upgrader.py

```diff
@@ -64,7 +64,15 @@ def load_last_run(self):
 
         self.last_run = None
 
-        self.last_run_file = os.path.join(sublime.packages_path(), 'User', 'Package Control.last-run')
+        if int(sublime.version()) < 3000:
+            cache_path = os.path.join(sublime.packages_path(), 'User')
+        else:
+            cache_path = os.path.join(sublime.cache_path(), 'Package Control')
+
+        if not os.path.isdir(cache_path):
+            os.mkdir(cache_path)
+
+        self.last_run_file = os.path.join(cache_path, 'Package Control.last-run')
```

Comment:
Might as well remove the "Package Control" part from the file name itself.

Reply:
I readded it because of `cache_path` pointing to User package on ST2. Would need to built the file name depending on ST version, too.